app/views/news_tematica/news_tematicas/_bloque_mas_leidos.html.haml

```diff
@@ -9,5 +9,4 @@
         %h5.titulo= link_to_con_estadisticas masleido.titulo, masleido.contenido_dominio_link, @news_tematica
         %p.resumen
           = raw masleido.descripcion
-          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>', masleido.contenido_dominio_link, @news_tematica,
-            class: 'leer_mas')
+          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>'.html_safe, masleido.contenido_dominio_link, @news_tematica, class: 'leer_mas')
```

Comment:
http://www.w3schools.com/tags/att_td_nowrap.asp says that `nowrap` is for table cells... I think that this is exactly the case in which `&nbsp;` should be applied, as it's defined as "non breaking space".

But even if you disagree, @d2bit is right in that on emails we must use very conservative HTML, which brings us again to the nbsp...